.github/dependabot.yml

```diff
@@ -10,8 +10,10 @@ updates:
     groups:
       kubernetes:
         patterns: [ "k8s.io/*", "sigs.k8s.io/*" ]
+        update-types: ["minor"]
```

Comment:
Should we set this for all the group and deps?

Reply:
these are the only 2 groups we currently have... unless we want to create more groups